proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -809,9 +735,7 @@ message GetValidatorParticipationRequest {
     }
 }
 
-// DEPRECATED: Prysm Web UI and associated endpoints will be fully removed in a future hard fork.
 message ValidatorParticipationResponse {
```

Comment:
we aren't using validator participating in webui anymore I think...

Reply:
all right, I will remove it

Reply:
unfortunately it's not marked as deprecated

Reply:
? the option deprecated = true got removed here , I don't understand why it's not marked as deprecated

Reply:
ahhh yes, will remove it